examples/kubernetes/static_provisioning/secret_authentication.yaml

```diff
@@ -0,0 +1,89 @@
+# Secret Authentication Example
+# This example demonstrates using a Kubernetes Secret to provide S3 credentials for the Mountpoint S3 CSI Driver.
+# This authentication method is particularly useful for:
+# 1. The user wants to set their own credentials which are different than the driver level credentials
+# 2. Using different credentials for different persistent volumes
+
+# First, create a Secret containing the S3 credentials
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-credentials
+  namespace: default
+type: Opaque
+data:
+  # Using base64 encoded values. Example:
+  # echo -n "ACCESS_KEY_ID" | base64
+  key_id: QUtJQVhYWFhYWFhYWFhYWFhY 
+  # echo -n "SECRET_ACCESS_KEY" | base64
+  access_key: U0VDUkVUWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWA==
+  
+  # You can also create the secret using kubectl:
+  # kubectl create secret generic s3-credentials \
+  #     --from-literal=key_id="ACCESS_KEY_ID" \
+  #     --from-literal=access_key="SECRET_ACCESS_KEY"
```

Comment:
Shouldn't this be `secret_access_key` ?

Reply:
the terminology we are following is `access_key` we might change it for all credentials eventually.
Drivcer level credentiials for the form from AWS is the same.

Reply:
Example: 
https://github.com/scality/mountpoint-s3-csi-driver/blob/main/charts/scality-mountpoint-s3-csi-driver/templates/node.yaml#L127
https://github.com/scality/mountpoint-s3-csi-driver/blob/main/charts/scality-mountpoint-s3-csi-driver/templates/node.yaml#L133